wasp/widgets.py

```diff
@@ -285,3 +285,80 @@ def touch(self, event):
         elif v >= self._steps:
             v = self._steps - 1
         self.value = v
+
+
+_message_string_x_coord = const(0)
+_message_string_y_coord = const(0)
+_yes_button_x_coord = const(20)
+_yes_button_y_coord = const(100)
+_no_button_x_coord = const(120)
+_no_button_y_coord = const(100)
```

Comment:
these coords will only really look right as long as we continue to only support 240x240 displays we may have to make this more dynamic in the future. But I just copied the patterns I've seen in other places and they're pretty hard coded